test/e2e/module-layer/pages/api/mixed.js

```diff
@@ -1,4 +1,4 @@
-import '../../lib/mixed-lib'
+// import '../../lib/mixed-lib'
 
```

Comment:
?

Reply:
It's was for the test `client packages in pages api`. seems we can remove the test and keep the import